js/menus/menuTemplate.js

```diff
@@ -254,6 +255,17 @@ function getTemplate(app) {
         }
     });
 
+    // Window menu -> bringToFront
```

Comment:
We might have to go with a different name for this. There is already an option called "Bring all to front" in the Window menu. Let's ask product about this.

Reply:
@KiranNiranjan Please change the verbatim to "Bring to Front on Notifications" in all the necessary places.

Reply:
Done 👍